src/borg/archiver/rinfo_cmd.py

```diff
@@ -48,9 +46,31 @@ def do_rinfo(self, args, repository, manifest, cache):
                     location=repository._location.canonical_path(),
                     version=repository.version,
                     append_only=repository.append_only,
-                    **info,
+                    encryption=info["encryption"],
                 )
             )
+
+            response = repository.info()
+            storage_quota = response["storage_quota"]
+            used = response["storage_quota_use"]
+
+            if storage_quota:
```

Comment:
just remove the "if" so it always is output.

of course you need some code to pretty output `storage_quota` for the "unlimited" case.